app/src/renderer/js/main.js

```diff
@@ -41,6 +45,70 @@ document.addEventListener('DOMContentLoaded', () => {
 
   const [micOnIcon, micOffIcon] = toggleAudioRecordBtn.children;
 
+  const createOption = (label, disabled = false) => {
```

Comment:
`disabled` here creates a [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap) when used. Use an options object instead.

Reply:
45c39dd